benchbuild/utils/actions.py

```diff
@@ -508,33 +514,31 @@ def __call__(self):
         self.status = StepResult.OK
         return results
 
-    def __str__(self, indent=0):
-        sub_actns = [a.__str__(indent + 1) for a in self.actions]
-        sub_actns = "\n".join(sub_actns)
+    def __str__(self, indent: int = 0) -> str:
+        sub_actns = "\n".join([a.__str__(indent + 1) for a in self.actions])
         return textwrap.indent("* All required:\n" + sub_actns, indent * " ")
 
 
 @attr.s
 class Containerize(RequireAll):
-    NAME = "CONTAINERIZE"
-    DESCRITPION = "Redirect into container"
+    NAME: str = "CONTAINERIZE"
+    DESCRITPION: str = "Redirect into container"
 
-    def requires_redirect(self):
+    def requires_redirect(self) -> bool:
         project = self.obj
         return not container.in_container() and (project.container is not None)
 
     @notify_step_begin_end
-    def __call__(self):
+    def __call__(self) -> None:
```

Comment:
R1710 (inconsistent-return-statements): Either all return statements in a function should return an expression, or none of them should.  <!-- Automatically posted by Lintly -->